emojify.el

```diff
@@ -1848,14 +1858,14 @@ completion UI to display properly emojis."
                                       minibuffer-setup-hook))
          (helm-after-initialize-hook (cons #'emojify--completing-read-helm-hook
                                            (bound-and-true-p helm-after-initialize-hook))))
-    (car (split-string (completing-read prompt
-                                        candidates
-                                        predicate
-                                        require-match
-                                        initial-input
-                                        hist
-                                        def
-                                        inherit-input-method)
+    (car (split-string (emojify-completing-read-function prompt
```

Comment:
Since `emojify-completing-read-function` is a variable it cannot be called directly, you will need to change this to `(funcall emojify-completing-read-function <args>)`

Reply:
Bah; good catch! Heh, too used to writing Scheme code…

Reply:
Yes, it does get confusing